codebase/sources/dhtmlxgantt.js

```diff
@@ -3609,8 +3610,13 @@ gantt._init_tasks_range = function(){
 	this._max_date = range.end_date;
 
 	if(!(this._max_date && this._max_date)){
-		this._min_date = new Date();
-		this._max_date = new Date(this._min_date);
+        if (this.config.ignore_max_date) {
```

Comment:
case:
    All tasks have no end date
    
![image](https://cloud.githubusercontent.com/assets/11240668/24835862/f56c2272-1d3f-11e7-9035-d27a2417f4c0.png)

expect:
 I still want to show the start date